tests/test_dim_reduction.py

```diff
@@ -0,0 +1,100 @@
+import unittest 
+import pandas as pd 
+import numpy as np 
+import os 
+import sys 
+current_path = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(current_path + "/../neasqc_wp61/models/quantum/")
+from dim_reduction import *
+
+class TestDimReduction(unittest.TestCase):
+    
+    def setUp(self):
```

Comment:
the dimension (here 768) should probably be an argument and not hard-coded. We want to re-use this for other embeddings which might be of different sizes.
Always make things generic when you can.